config/purposes/scrna_core_cell_extraction.yml

```diff
@@ -62,30 +62,25 @@ LRC PBMC Bank:
       # number of source wells to pool into a single destination well by sample from LRC Bank to LRC PBMC Bank
       # e.g. 6 copies of sample in 6 wells, '2' in the config, pool to make 3 child wells
       number_of_source_wells: 2
+  :presenter_class: Presenters::BankingPlatePresenter
   :label_template: plate_quad_qc # creates QC4 barcodes
   :file_links:
     - name: 'Download Hamilton LRC Blood Bank plate to LRC PBMC Bank plate CSV'
       id: hamilton_lrc_blood_bank_to_lrc_pbmc_bank
     - name: 'Download Hamilton LRC PBMC Bank plate to LRC Bank Seq and Spare tubes CSV'
       id: hamilton_lrc_pbmc_bank_to_lrc_bank_seq_and_spare
+      # states can be specified as string, symbol, or array values to set the
+      # includes or hash with includes/excludes keys and string, symbol, or
+      # array values. In this particular case, we want to enable the download
+      # link only when the plate is in the passed and qc_complete states.
```

Comment:
```
states can be specified as string, symbol, or array values to set the
includes or hash with includes/excludes keys and string, symbol, or
array values.
```
This is generic documentation - could go in `docs/purposes_yaml_files.md` ?

```
In this particular case, we want to enable the download link only when the plate is in the passed and qc_complete states.
```
I think this is obvious from the code but leave it in if you like as I obviously have more Limber context than some future developers might!